doc/guides/05-how_to_use_interactions.md

```diff
@@ -0,0 +1,1470 @@
+# How to Use Interactions:
```

Comment:
This file is too long unfortunately - we can defer to Jo on how to break it up when we meet tomorrow